src/menu/examples.js

```diff
@@ -226,4 +228,48 @@ export default {
       />
     );
   },
+
+  [examples.LONG_LIST]: function LongListStory() {
+    const items = [...new Array(1500)].map((_, index) => ({
+      label: `item number: ${index + 1}`,
+    }));
+
+    function VirtualList(props) {
+      return (
+        <div ref={props.$ref}>
+          <List
+            role={props.role}
+            height={500}
+            rowCount={props.children.length}
+            rowHeight={36}
+            rowRenderer={({index, key, style}) => (
+              <OptionList
+                key={key}
+                style={style}
+                {...props.children[index].props}
+                overrides={{
+                  ListItem: {
+                    style: {
+                      paddingTop: 0,
+                      paddingBottom: 0,
+                      display: 'flex',
+                      alignItems: 'center',
+                    },
+                  },
+                }}
+              />
+            )}
+            width={200}
```

Comment:
Let's not set the width here but let it extend to the control width. Also there lost box shadow on the dropdown, not sure if it's width related.